raiden-ts/tests/integration/integration.spec.ts

```diff
@@ -73,13 +73,12 @@ describe('integration', () => {
   });
 
   afterAll((done) => {
-    raiden.stop();
     raiden.events$.pipe(filter((value) => value.type === 'raiden/shutdown')).subscribe(done);
+    raiden.stop();
   });
 
   test('account is funded', async () => {
-    const balance = await raiden.getBalance(raiden.address);
-    expect(balance.eq(bigNumberify(ethBalance))).toBe(true);
+    await expect(raiden.getBalance(raiden.address)).resolves.toEqual(bigNumberify(ethBalance));
```

Comment:
What was actually the issue here? Are we sure this is formatted? Looks preeeetty long this line and I actually liked the two lines before.

Reply:
This is actually intended to have any thrown exception by `getBalance` as part of the error output, so it makes the error case more readable. The same with switching to a `toEqual` comparison, so it should show up as `rejected instead of resolved` and both values in case of mismatch, instead of just `expected true, got false`.